docs/admin/system_admin/multiple_nodes.md

```diff
@@ -15,39 +15,32 @@ This feature currently works only on Linux. Other prerequisites for activating i
 
 This is advanced feature aimed at existing LeoFS users who need to run another cluster or more gateways using the same hardware; it is recommended to have a working cluster where all work and log directories are configured to use non-default locations before enabling this.
 
-## Activating more instances of the same node
+## Preparing more instances of the same node
 
 Let's say we have LeoStorage instance running through systemd (`leofs-storage` service) and currently it loads configuration from `/etc/leofs/leo_storage`. We want to run another instance on the same system; it needs separate name. In this example, we'll call it "second". To create such instance:
 
-1. Create local copy of systemd unit file with new name: `cp /usr/lib/systemd/system/leofs-storage.service /etc/systemd/system/leofs-storage-second.service`
+1. Create local copy of systemd unit file with new name
+    - EL7&nbsp;version: `cp /usr/lib/systemd/system/leofs-storage.service /etc/systemd/system/leofs-storage-second.service`.
+    - Ubuntu&nbsp;16.04&nbsp;version: `cp /lib/systemd/system/leofs-storage.service /etc/systemd/system/leofs-storage-second.service`.
 2. Edit `/etc/systemd/system/leofs-storage-second.service`, changing `Environment=NODE_EXTRA_NAME=` line to `Environment=NODE_EXTRA_NAME=second`.
-3. Reload systemd configuration: `systemctl daemon-reload`
+3. Reload systemd configuration: `systemctl daemon-reload`.
+
+First step here depends on distribution because default location of systemd unit file can differ; it can be checked in "UNIT FILE LOAD PATH" section of systemd.unit(5) man page.
```

Comment:
Add the external link for systemd.unit(5) on Related Links.
- For debian: https://manpages.debian.org/jessie/systemd/systemd.unit.5.en.html#UNIT_LOAD_PATH

I can't find the one for EL7 so please add if you know the appropriate one.